src/pages/contacts/Create.tsx

```diff
@@ -0,0 +1,148 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import { ChevronDownIcon, ChevronLeftIcon, SearchIcon } from "lucide-react";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import {
+  Select,
+  SelectGroup,
+  SelectValue,
+  SelectTrigger,
+  SelectContent,
+  SelectItem,
+} from "@/components/ui/select";
+import routes from "@/constants/routes";
+
+function Header() {
+  return (
+    <div className="flex flex-col gap-5 w-full">
+      <button className="flex items-center justify-center w-8 h-8 bg-[#1B0F00]/20 rounded-full border-[1px] border-[#1B0F00]/6">
+        <ChevronLeftIcon width={16} strokeWidth={1} color="#1B0F00" />
+      </button>
+      <div className="flex justify-between items-center">
+        <h1 className="text-text-300 text-xl font-medium">
+          <FormattedMessage
+            id="pages.createContact.header.title"
+            defaultMessage="New contact"
+            description="Title for create contact page"
+          />
+        </h1>
+
+        <Button
+          className="gap-1 w-fit text-text-300 bg-transparent text-sm font-medium p-0 border-none hover:bg-transparent"
+          variant="outline"
+        >
+          <SearchIcon className="w-4 h-4 text-text-300" />
+          <FormattedMessage
+            id="pages.createContact.search"
+            defaultMessage="Search contacts"
+            description="Search contacts button"
+          />
+        </Button>
+      </div>
+    </div>
+  );
+}
+
+function Form() {
+  const [isExtraFieldsVisible, setIsExtraFieldsVisible] = useState(false);
+
+  return (
+    <div className="flex-1 flex flex-col gap-3 w-full">
+      <Input id="name" label="Name" required />
+      <Input id="email" label="Email address" required />
+      <Input id="address" label="Postal address" required />
+
+      <Button
+        className={`gap-1 w-fit text-text-200 bg-transparent text-sm font-medium p-0 border-none hover:bg-transparent mx-auto ${
+          isExtraFieldsVisible ? "hidden" : "flex"
+        }`}
+        variant="outline"
+        onClick={() => setIsExtraFieldsVisible(!isExtraFieldsVisible)}
+      >
+        <FormattedMessage
+          id="pages.createContact.viewAllFields"
+          defaultMessage="View more"
+          description="View more fields button"
+        />
+        <ChevronDownIcon className="w-4 h-4 text-text-200" />
+      </Button>
+
+      <div
+        className={`flex flex-col gap-3 ${
+          isExtraFieldsVisible ? "block" : "hidden"
+        }`}
+      >
+        <Input id="dateOfBirth" label="Date of birth (MM/DD/YYYY" />
+        <Select>
+          <SelectTrigger label="Country of issuing" id="countryOfIssuing">
+            <SelectValue placeholder="Select an option" />
```

Comment:
Should placeholders already be localized with `FormattedMessage`?

```suggestion
            <SelectValue placeholder={<FormattedMessage
            id="pages.createContact.select.country"
            defaultMessage="Select an option"
            description="Placeholder for country selection"
          />} />
```

This applies to other non-translated fields/placeholders/etc. in this file.

Reply:
@mjkeaton we do changes in our i18n format now. We use other id format. Check https://github.com/BitcreditProtocol/E-Bill-frontend/pull/85

Reply:
Good catch, thank you! I'm gonna replace that. In this case, we must use the imperative API so we don't have to change the typedef for the component.

Ref: https://formatjs.github.io/docs/react-intl/api/#useintl-hook